package.json

```diff
@@ -24,7 +24,7 @@
     "docs:build": "lerna run check-readme && lerna run docs:build && ./scripts/prepare-docs.sh",
     "docs:lint": "prettier '**/*.md' --check",
     "docs:lint:fix": "prettier '**/*.md' --write",
-    "docs:serve": "mkdocs serve --watch docs/pages --config-file docs/mkdocs.yml",
+    "docs:serve": "mcd docs; yarn start",
```

Comment:
I think we should remove this script.  Feels weird to have to build from inside docs but can run serve from here.